fdbrpc/boost/thread/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+set(SRCS pthread/thread.cpp
```

Comment:
Should pthread/thread.cpp be included for WIN32?

Reply:
Probably not... Though I copied this code from the boost-cmake repository. I think a cleaner way of doing this is to change the cmake code to compile thread and context in `ExternalProject_add` (copying the files into the repository seems dangerous and painful)